spec/lib/twingly/url_spec.rb

```diff
@@ -135,6 +134,13 @@ def leading_and_trailing_whitespace
       end
     end
 
+    context "downcases the protocol" do
```

Comment:
"downcases the protocol" sounds more like an "it" than an "context". I'd go with something like:

```ruby
context "when given an URL with uppercase protocol" do
  it "downcases the protocol" do
    ...
```

Reply:
I just copied an existing spec! 🙈

Reply:
I agree :-) I also think we should use "scheme" instead of "protocol", I can fix both tomorrow

Reply:
Hmm, after I've looked more closely I think it is a bigger task to use the style you suggest, because there's soo many specs that should be changed. I think if we start mixing, it might make the situation worse.

Reply:
I think the style I suggested is very prominent in the file? Are we thinking about the same thing?

Reply:
For example: https://github.com/twingly/twingly-url/blob/c2218e98ff0892e63adc483f4cf1c06e4f01f3ae/spec/lib/twingly/url_spec.rb#L102-L156 is in that style

I think the other style has snuck in lately

Reply:
I gave it a go anyway, but only for the new spec

Reply:
Ah, I see your change now.